requirements/requirements.txt

```diff
@@ -44,79 +44,81 @@ django-taggit==0.22.2
 django==1.11.20
 djangorestframework-yaml==1.0.3
 djangorestframework==3.7.7
+docutils==0.14            # via python-daemon
 future==0.16.0            # via django-radius
-hyperlink==18.0.0         # via twisted
-idna==2.8                 # via hyperlink, requests
+hyperlink==19.0.0         # via twisted
+idna==2.8                 # via hyperlink, requests, twisted
 incremental==17.5.0       # via twisted
 inflect==2.1.0            # via jaraco.itertools
 irc==16.2
 isodate==0.6.0            # via msrest, python3-saml
 jaraco.classes==2.0       # via jaraco.collections
-jaraco.collections==2.0   # via irc, jaraco.text
+jaraco.collections==2.0   # via irc
 jaraco.functools==2.0     # via irc, jaraco.text, tempora
-jaraco.itertools==4.4.1   # via irc
+jaraco.itertools==4.4.2   # via irc
 jaraco.logging==2.0       # via irc
 jaraco.stream==2.0        # via irc
-jaraco.text==2.0          # via irc, jaraco.collections
+jaraco.text==3.0          # via irc, jaraco.collections
 jinja2==2.10.1
 jsonpickle==1.1           # via asgi-amqp
 jsonschema==2.6.0
-kombu==4.2.1              # via asgi-amqp, celery
-lxml==4.3.1               # via xmlsec
+kombu==4.5.0              # via asgi-amqp, celery
+lockfile==0.12.2          # via python-daemon
+lxml==4.3.3               # via xmlsec
 markdown==2.6.11
-markupsafe==1.1.0         # via jinja2
-more-itertools==6.0.0     # via irc, jaraco.functools, jaraco.itertools
+markupsafe==1.1.1         # via jinja2
+more-itertools==7.0.0     # via irc, jaraco.functools, jaraco.itertools
 msgpack-python==0.5.6     # via asgi-amqp
-msrest==0.6.4             # via azure-keyvault, msrestazure
+msrest==0.6.6             # via azure-keyvault, msrestazure
 msrestazure==0.6.0        # via azure-keyvault
 netaddr==0.7.19           # via pyrad
-oauthlib==2.0.6           # via django-oauth-toolkit, requests-oauthlib, social-auth-core
+oauthlib==3.0.1           # via django-oauth-toolkit, requests-oauthlib, social-auth-core
 ordereddict==1.1
 pexpect==4.6.0
-pkgconfig==1.4.0          # via xmlsec
+pkgconfig==1.5.1          # via xmlsec
 prometheus_client==0.6.0
 psutil==5.4.3
 psycopg2==2.7.3.2
 ptyprocess==0.6.0         # via pexpect
-pyasn1-modules==0.2.4     # via python-ldap, service-identity
+pyasn1-modules==0.2.5     # via python-ldap, service-identity
 pyasn1==0.4.5             # via pyasn1-modules, python-ldap, service-identity
 pycparser==2.19           # via cffi
 pygerduty==0.37.0
 pyhamcrest==1.9.0         # via twisted
 pyjwt==1.7.1              # via adal, social-auth-core, twilio
-pyopenssl==19.0.0         # via service-identity
+pyopenssl==19.0.0         # via twisted
 pyparsing==2.2.0
 pyrad==2.1                # via django-radius
 pysocks==1.6.8            # via twilio
-python-daemon==2.2.0      # via ansible-runner
+python-daemon==2.2.0
```

Comment:
ansible-runner doesn't pull this in now?

Reply:
The automated tool will not add this comment, and I do not know why. Yes, it is listed as a dependency, so I'm not sure what's up with that.

Reply:
Got it, that's because it was put into `requirements/requirements.in`. Since it's a top-level dependency, it is not annotated as being a dependency of another library here.

Here is a change to resolve this, and the output of the tool along with it:

```diff
diff --git a/requirements/requirements.in b/requirements/requirements.in
index 05853a486f..747d18b079 100644
--- a/requirements/requirements.in
+++ b/requirements/requirements.in
@@ -35,7 +35,6 @@ psutil==5.4.3
 psycopg2==2.7.3.2   # problems with Segmentation faults / wheels on upgrade
 pygerduty==0.37.0
 pyparsing==2.2.0
-python-daemon==2.2.0
 python-dateutil==2.7.2  # contains support for TZINFO= parsing
 python-memcached==1.59
 python-radius==1.0
diff --git a/requirements/requirements.txt b/requirements/requirements.txt
index 10254944b3..0ae05d9538 100644
--- a/requirements/requirements.txt
+++ b/requirements/requirements.txt
@@ -2,7 +2,7 @@
 # This file is autogenerated by pip-compile
 # To update, run:
 #
-#    pip-compile -U -r --allow-unsafe --output-file requirements/requirements.txt requirements/requirements.in
+#    pip-compile --allow-unsafe --output-file=requirements/requirements.txt requirements/requirements.in
 #
 adal==1.2.1               # via msrestazure
 amqp==2.4.2               # via kombu
@@ -15,7 +15,7 @@ asn1crypto==0.24.0        # via cryptography
 attrs==19.1.0             # via automat, service-identity, twisted
 autobahn==19.3.3          # via daphne
 automat==0.7.0            # via twisted
-azure-common==1.1.19      # via azure-keyvault
+azure-common==1.1.20      # via azure-keyvault
 azure-keyvault==1.1.0
 azure-nspkg==3.0.2        # via azure-keyvault
 billiard==3.6.0.0         # via celery
@@ -90,7 +90,7 @@ pyopenssl==19.0.0         # via twisted
 pyparsing==2.2.0
 pyrad==2.1                # via django-radius
 pysocks==1.6.8            # via twilio
-python-daemon==2.2.0
+python-daemon==2.2.3      # via ansible-runner
 python-dateutil==2.7.2
 python-ldap==3.2.0        # via django-auth-ldap
 python-memcached==1.59
```